pkg/util/file/file.go

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
You can use this package instead https://github.com/kubernetes/utils/tree/master/path